app/views/categories/index.html.erb

```diff
@@ -0,0 +1,50 @@
+<body>
+  <main>
+    <h2>Products by Category</h2>
+
+  <section class= "product-list">
+  <h4>Annual</h4>  
+    <% @product.categories each do |annual| %>   
+    <h4><%= product.name.capitalize %></h4>
+      <ul>
+        <li>Description: <%= product.description.capitalize %></li>
+        <li>Price: $<%= product.price %></li>
+        <li>Stock: <%= product.stock %> units</li>
+        <li>Available: <%= product.available %></li>
+        <li>User_id: <%= product.user_id %></li>
+        <li>photo_url</li>
+      </ul>
+    <%end%>
+  </section>
+
+  <section class= "product-list">
+  <h4>Flower</h4>  
```

Comment:
Ugh! I need to get 'categories' up and running so that we can pass each category into the method so that this section is more flexible and DRY-er. Looks good, though!

Reply:
I'd love to work together on that logic because product and category are very attached.